File: VariablesConstantsbucle/fase3.js
var refYear = 1948;
var myYear = 1978;
var isLeap = false;

for (i = refYear; i <= myYear; i++) {
    if ((i % 4 == 0) && ((i % 100 != 0) || (i % 400 == 0))) {
        isLeap = true;
    } else {
        isLeap = false;
    }

    if (isLeap) {
        console.log(i + " is a leap year");
    } else {
        console.log(i + " is a normal year");
    }
}

File: VariablesConstantsbucle/Fase3.java

public class Fase3 {

	public static void main(String[] args) {
		int refYear = 1948;
		int myYear = 1978;
		boolean isLeap = false;

		for (int i = refYear; i <= myYear; i++) {
			if ((i % 4 == 0) && ((i % 100 != 0) || (i % 400 == 0))) {
				isLeap = true;

			} else {
				isLeap = false;

			}
			if (isLeap == true) {
				System.out.println(i + " is a leap year");
			} else {
				System.out.println(i + " is a normal year");
			}
		}

	}
}
